README.md

```diff
@@ -15,6 +15,7 @@ As solving the above issue requires addition of various vulnerabilities, hence i
 - Spring Boot
 - ReactJS
 - Javascript/TypeScript
+- OpenAI
```

Comment:
It would be helpful to provide a brief description or a link to OpenAI's documentation next to the OpenAI technology addition. This would help unfamiliar readers understand what OpenAI is and how it is used in this project. [medium]